modules/install-creating-aws-local-zones-vpc-stack.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: PROCEDURE
+[id="install-aws-local-zones-vpc-stack_{context}"]
+=== Creating a VPC for AWS Local Zones
+
+Create a virtual private cloud (VPC) to use with AWS Local Zones.
+
+.Procedure
+// TODO: GitHub links are not okay.
```

Comment:
On the original document we are using the Template directly from the documentation, you must update it on this PR based on the installer repo (downloaded on the steps below)

- VPC : https://57427--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone.html#installation-cloudformation-vpc-localzone_installing-aws-localzone
- LZ subnet: https://57427--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone.html#installation-cloudformation-subnet-localzone_installing-aws-localzone

Reply:
TODO